pom.xml

```diff
@@ -141,10 +141,17 @@
         <dependency>
             <groupId>com.powsybl</groupId>
             <artifactId>powsybl-dynawaltz</artifactId>
+            <version>2.5.0-alpha-1</version>
         </dependency>
         <dependency>
             <groupId>com.powsybl</groupId>
             <artifactId>powsybl-dynawaltz-dsl</artifactId>
+            <version>2.5.0-alpha-1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.gridsuite</groupId>
+            <artifactId>gridsuite-filter</artifactId>
+            <version>1.0.8</version>
```

Comment:
1.0.9 is available now

Reply:
DONE in https://github.com/gridsuite/dynamic-simulation-server/pull/95/commits/9c1f3283878c39cac8535ed7157cc5e04ff81e14